addons/l10n_in_edi_ewaybill/views/res_config_settings_views.xml

```diff
@@ -27,6 +28,12 @@
                             </div>
                             <div class='mt8'>
                                 <button name="l10n_in_edi_ewaybill_test" icon="fa-arrow-right" type="object" string="Verify Username and Password" class="btn-link"/>
+                                <div class="mx-3 badge rounded-pill text-white bg-success border border-success" role="img" attrs="{'invisible': [('l10n_in_edi_ewaybill_token_status', '=', False)]}">
```

Comment:
Can not this be confusing?  It could say disconnected when the system can connect automatically.  Why not put a cr.commit()  raise UserError(_('Connection successfull!") in the Verify link?

Reply:
we can send messages with sticky notes without raising but the objective is not only for the user to give information.
but whenever any admin comes to the setting then he knows that it's set up and working.
we can also add a disconnected logo so users are not confused.

Reply:
@jva-odoo It looks like a connection the user is having all the time (like a cable that is plugged in or not), while it is just a token that might need refreshing and normally, if the first time, it was ok, it should be ok for the other times.  Won't he see disconnected when the token just needs refreshing as well thinking something is wrong while it is not?

Reply:
hmm, right because token refresh when EDI is called